floris/simulation/flow_field.py

```diff
@@ -69,6 +76,40 @@ def wind_directions_validator(self, instance: attrs.Attribute, value: NDArrayFlo
         """Using the validator method to keep the `n_wind_directions` attribute up to date."""
         self.n_wind_directions = value.size
 
+    @het_config.validator
+    def het_config_validator(self, instance: attrs.Attribute, value: dict | None) -> None:
+        """Using the validator method to check that the het_config dictionary has the correct
+        key-value pairs.
+        """
+        if value is not None:
```

Comment:
@RHammond2 would value ever be None here?

Reply:
If no `het_config` is passed, then yes, because `None` is the default value. Maybe the question is could the control flow be switch to do the following to clean up the indentation?
```python
if value is None:
    return

# code validation
````

Reply:
Thanks for the suggestion to early-exit @RHammond2.